site/docs/main/configuration.md

```diff
@@ -189,6 +190,34 @@ If present, the value of the `CONTOUR_NAMESPACE` environment variable is used as
 
 The `CONTOUR_NAMESPACE` environment variable is set via the [Downward API][6] in the Contour [example manifests][7].
 
+## Bootstrap Config File
+
+The bootstrap configuration file is generated by an initContainer in the Envoy daemonset which runs the `contour bootstrap` command to generate the file.
+This configuration file configures the Envoy container to connect to Contour and receive configuration via xDS.
+
+The next section outlines all the available flags that can be passed to the `contour bootstrap` command which are used to customize
+the configuration file to match the environment in which Envoy is deployed. 
+
+### Flags
+
+There are flags that can be passed to `contour boostrap` that help configure how Envoy
+connects to Contour:
+
+| Flag | Default  | Description |
+|------------|----------|-------------|
+| <nobr>--resources-dir</nobr> | "" | Defines the directory where resource files will be written.  |
+| <nobr>--admin-address</nobr> | 127.0.0.1 | Defines the address the Envoy admin webpage will listen on.  |
+| <nobr>--admin-port</nobr> | 9001 | Defines the port the Envoy admin webpage will listen on.  |
+| <nobr>--xds-address</nobr> | 127.0.0.1 | Defines the address to connect to Contour xDS server on.  |
+| <nobr>--xds-port</nobr> | 8001 | Defines the port to connect to Contour xDS server on. |
+| <nobr>--envoy-cafile</nobr> | "" | Defines the gRPC CA filename for Envoy to load.  |
```

Comment:
Nit: Maybe something in here to say that the gRPC being talked about is xDS? Or just replacing gRPC with xDS? Similarly for all of the cert flags.
```suggestion
| <nobr>--envoy-cafile</nobr> | "" | Defines the xDS CA filename for Envoy to load.  |
```

Reply:
Yeah can do that, I just copied what the flags have today as help: https://github.com/projectcontour/contour/blob/1437a9dc33fc0ac81ba966cf258433c891725ece/cmd/contour/bootstrap.go#L33

Reply:
I added https://github.com/projectcontour/contour/pull/3083 to update those flags, I'll do the docs here.